src/index.js

```diff
@@ -19,7 +21,11 @@ import {
 
 import { MarcAuthoritiesAppContext } from './components';
 
+import commands from './commands';
+import KeyShortcutsWrapper from './components/KeyShortCutsWrapper/KeyShortCutsWrapper';
```

Comment:
You don't have to write `KeyShortCutsWrapper` twice when importing, since you have an `index.js` inside `KeyShortCutsWrapper` folder which exports the component and is accessed by default. Moreover, you can import `KeyShortcutsWrapper` directly from the `components` folder like other components as there's also an `index.js`.

```suggestion
import {
  KeyShortcutsWrapper,
  MarcAuthoritiesAppContext,
} from './components';

import commands from './commands';
```

You'll need to change a `src\components\KeyShortCutsWrapper\index.js` file a bit too for this to work. Look at the `index.js` in other components for the reference.

Also, please note that you have two different versions of the 'KeyShortcutsWrapper' spelling: 'KeyShortcutsWrapper' and 'KeyShortCutsWrapper'. It would be better to make them identical across the module to eliminate the possible confusion.